### PR TITLE
fix(pci-gateway): Missing style from pci-common

### PR DESCRIPTION
**Pull request by Tsiorifamonjena**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop` 
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #TAPC-2377
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

**Review by SimonChaumet — APPROVED** [review]

**Review by chipp972 — APPROVED** [review]

**Review by tristanwagner — DISMISSED** [review]

File: packages/manager/apps/pci-gateway/package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "@ovh-ux/manager-config": "^8.0.1",
     "@ovh-ux/manager-core-api": "^0.9.0",
     "@ovh-ux/manager-pci-common": "^0.11.2",
-    "@ovh-ux/manager-react-components": "^1.41.1",
+    "@ovh-ux/manager-react-components": "^1.41.2",
```
Comment (fredericvilcot) [inline packages/manager/apps/pci-gateway/package.json:21]:
```suggestion
    "@ovh-ux/manager-react-components": "^1.43.0",
```

**Review by fredericvilcot — CHANGES_REQUESTED** [review]

**Comment by fredericvilcot** [comment]
@Tsiorifamonjena 
Please rebase your branch from develop 🙏

**Review by fredericvilcot — APPROVED** [review]

**Review by selm3n — APPROVED** [review]

**Review by SimonChaumet — APPROVED** [review]

File: packages/manager/apps/pci-gateway/tailwind.config.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ module.exports = {
   content: [
     './src/**/*.{js,jsx,ts,tsx}',
     '../../../manager-react-components/src/**/*.{js,jsx,ts,tsx}',
+    '../../modules/manager-pci-common/**/*.{js,jsx,ts,tsx}',
```
Comment (aboungnaseng-ovhcloud) [inline packages/manager/apps/pci-gateway/tailwind.config.js:9]: @fredericvilcot  are we sure about that ?

Comment (aboungnaseng-ovhcloud) [reply packages/manager/apps/pci-gateway/tailwind.config.js:9]: No sorry 👍  Its seems to be OK

**Review by aboungnaseng-ovhcloud — APPROVED** [review]

File: packages/manager/apps/pci-gateway/tailwind.config.js (hunk 2 above)
Comment (fredericvilcot) [reply packages/manager/apps/pci-gateway/tailwind.config.js:9]: @aboungnaseng-ovhcloud 
yes, it's mandatory to import tailwind style from `manager-pic-common`, if not UI is broken.

**Review by acavalloni — APPROVED** [review]

**Pull request merged**